SignalShareExtension/SAELoadViewController.swift

```diff
@@ -39,6 +68,16 @@ class SAELoadViewController: UIViewController {
         self.view.addSubview(activityIndicator)
         activityIndicator.autoCenterInSuperview()
 
+        progressView = UIProgressView(progressViewStyle: .default)
```

Comment:
`OWSProgressView` for consistency?  `UIProgressView` is too minimal to use in the contexts where we already use `OWSProgressView`.

Reply:
Note that `OWSProgressView` requires you to `autoSetDimension` its size.